maven-artifact/src/main/java/org/apache/maven/artifact/versioning/ComparableVersion.java

```diff
@@ -354,15 +354,15 @@ public String toString()
         implements Item
     {
         private static final List<String> QUALIFIERS =
-                Arrays.asList( "alpha", "beta", "milestone", "rc", "snapshot", "", "sp"  );
+                Arrays.asList( "alpha", "beta", "milestone", "preview", "rc", "snapshot", "", "sp"  );
```

Comment:
What is the difference between preview and release candidate?

Reply:
preview is used like milestone, so previous to RC

Reply:
![image](https://user-images.githubusercontent.com/5782559/197350416-6d9a14c5-4f81-4450-a0c3-dae1d617b675.png)